hive/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -130,6 +130,7 @@ protected void doRefresh() {
     }
 
     refreshFromMetadataLocation(metadataLocation);
+    LOG.debug("Refreshed [{}]", fullName);
```

Comment:
There is already a log in `BaseMetastoreTableOperations` for this. It has the location, but not the table name. Maybe just add table name to that one.

Reply:
removed